cpp/example/padmm_random_coverage.cc

```diff
@@ -129,24 +130,23 @@ int main(int, char **) {
   // sopt::logging::set_level("debug");
   //  purify::logging::set_level("debug");
   const std::string &name = "M31";
-  const t_real FoV = 1;       // deg
-  const t_real max_w = 100.;  // lambda
-  const t_real snr = 10;
+  const t_real FoV = 15;       // deg
+  const t_real max_w = 15.;  // lambda
+  const t_real snr = 30;
   const bool w_term = false;
   const std::string kernel = "kb";
   std::string const fitsfile = image_filename(name + ".fits");
   auto M31 = pfitsio::read2d(fitsfile);
-  const t_real cellsize = 1;  // FoV / M31.cols() * 60. * 60.;
+  const t_real cellsize = FoV / M31.cols() * 60. * 60.;
```

Comment:
Is this correct? if it's so, proably I would find it more readable as: 
```suggestion
  const t_real cellsize = (FoV * 60. * 60.)/ 31.cols(); // [arcsecs/px]
```

Reply:
Actually, I think this formula is only right for small fields of view (FoV). I wrote a function that will work though...